code/modules/vending/snack.dm

```diff
@@ -51,12 +51,16 @@
 
 /obj/machinery/vending/snack/blue
 	icon_state = "snackblue"
+	allow_custom = FALSE
```

Comment:
Do the inverse. Instead of setting FALSE everywhere just make the default value FALSE and set it TRUE where needed

Reply:
I considered that option but my thinking was most vendors (and the default behavior of new ones) should be to allow it, you only really want it disallowed for special vendors like off-station/ruin ones, and subtypes that aren't really separate vendors